kotsadm/pkg/store/s3pg/license_store.go

```diff
@@ -85,3 +85,19 @@ func (s S3PGStore) GetLicenseForAppVersion(appID string, sequence int64) (*kotsv
 
 	return nil, nil
 }
+
+func (s S3PGStore) CheckDoesLicenseExists(license string) (string, error) {
+	db := persistence.MustGetPGSession()
+	query := `select slug from app where license = $1`
```

Comment:
This is not going to work if license is updated or even re-downloaded.  You have to match license ID only.